README.md

```diff
@@ -65,6 +65,94 @@ GitHub Projects es una herramienta integrada en la plataforma GitHub que te perm
 > [!TIP]
 > Te compartimos un [▶️video](https://youtu.be/r8jQ9hVA2qs?si=IRzStEVNR_UrMVpZ) para entender mejor estos conceptos.
 
+
+<!-- 
+  <<< Comandos Git >>> 
+-->
+<details id=1>
+<summary><h2>Comandos básicos de Git</h2></summary>
+
+**Configurar un usuario**
+
+Puedes configurar el nombre del autor y la dirección de correo electrónico para registrar de quien provienen los cambios.
+
+`git config --global user.name "Nombre Usuario"`
+
+`git config --global user.email nombreusuario@gmail.com`
+
+**Crear un repositorio local**
+
+Cuando quieres crear un repositorio local primero debes inicializarlo con el siguiente comando:
+
+`git init`
+
+**Clonar un repositorio**
+
+Para crear o clonar una copia de otro repositorio local utilizamos:
+
+`git clone /ruta/del/repositorio`
+
+**Agregar archivos**
+
+Si queremos agregar archivos al stage o index (para luego hacer commit) lo hacemos con el comando add:
+
+`git add <nombre_archivo`
```

Comment:
@aleepsy I think `>` is missing here.

`git add <nombre_archivo>`

Reply:
Fixed.